spec/fixtures/vcr/_page_count_returns_total_count.yml

```diff
@@ -2,7 +2,7 @@
 http_interactions:
 - request:
     method: get
-    uri: https://pdc.host.dev.eng.pek2.redhat.com/rest_api/v1/arches/?page=2
+    uri: https://test.com/rest_api/v1/arches/?page=2
```

Comment:
Thought I had explained this before, but I think `test.com` is a poor choice here. If you want to pick a demo URL, you should use a reserved domain name like [.test](https://en.wikipedia.org/wiki/.test) or similar.

Having said that tho, perhaps that is not a good choice either since PDC is not hosted under the example domain. So maybe just choose Fedoras PDC as an example?

Reply:
Do you mean to use .test to instead test.com?
I am not sure if we are on the same page about why I didn't use the fedora url.
In fedora pdc, the releases endpoint return these below items:

Reply:
I was not aware that we also have an internal 'plugin?' for PDC which adds these fields. IMHO this gem has to follow the same setup as the upstream product-definition-center. This gem works with upstream while you'd need an internal part/gem which adds functionality for these fields. That way you'd avoid the confusion/perhaps leakage of internal data and can refer to Fedoras PDC in your test data in this gem.

Reply:
Btw. I do realise that making these changes will slow you down from just moving forward. However I do think it's the right choice to keep upstream upstream. Perhaps the decision to introduce these fields in product definition center by keeping them separate was not wise since it leads to additional fragmentation, but to me it means that it solves the problem you're facing with leaking internal data.

Reply:
As I know, this package is just used by et, not by others, you can see the activesupport is 3.2 here, like which et is using. So I think we should keep codes in one gem.
And also for me I prefer we use the .test here as they are testcases, then we can do other works after this change. What's your thinking?

Reply:
Dear @xiangge if you want to keep changes small for this patch, that's fine by me. So use the `.test` top level domain if you want.

I think in the long run, the current test data exposes internal identifiers as I have mentioned on another patch I'm not comfortable with. If you have an idea to address this by not splitting it up, I'm all ears. I'm sure it can be done either way without too much work involved.

Reply:
@romanofski I change the "test.com" to "test" here, please help to review, if this is also incorrect, would you please give me an example?

For the question of internal identifiers, like what I mentioned in that patch, I can make another patch to change the test data to fedora's and also manually add the missing columns, then we can solve this problem.